share/getters/cascade.go

```diff
@@ -37,7 +38,12 @@ func (cg *CascadeGetter) GetShare(ctx context.Context, root *share.Root, row, co
 		attribute.Int("col", col),
 	))
 	defer span.End()
-
+	if row >= len(root.RowRoots) || col >= len(root.ColumnRoots) {
```

Comment:
When can this happen? 🤔

Reply:
Thought the rpc

`celestia rpc share GetShare "$(celestia rpc header GetByHeight 101810 | jq '.result.dah' -r)" 101810 0`

101810 -> rowIndex
0 -> colIndex

Reply:
Ah, so the rpc request is malformed

Reply:
Yes